docs/api/k-tagged-template.md

```diff
@@ -0,0 +1,42 @@
+# `k` tagged template
```

Comment:
This documentation could be improved by showing full working examples. See [the useCss documentation](https://kremling.js.org/api/use-css.html) for what I mean.

One concrete reason why this would help is because the current documentation does  not show you how  to import the `k` tagged template.

A full working example would be something like this:

### Basic Example
```js
import {k} from 'kremling'

function Foo(props) {
  const css  = useCss(css);

  return <button {...scope} className="foo">Foo</button>
}

// You don't need `&` in your css if you're using kremling-inline-loader and the `k` tag
const css = k`
.foo {
  background-color: lawngreen;
}
`
```

Reply:
good call